digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java

```diff
@@ -848,7 +848,7 @@ public boolean setDoneState(long taskId, TaskStateCode beforeState, TaskStateCod
             return n > 0;
         }
 
-        public boolean setDoneStateShortCircuit(long taskId, TaskStateCode beforeState, TaskStateCode afterState, Config error)
+        public boolean setErrorStateShortCircuit(long taskId, TaskStateCode beforeState, TaskStateCode afterState, Config error)
```

Comment:
oh right. Merging #310 first was my mistake.
#310 was to this PR because it caused conflicts.